vue/package.json

```diff
@@ -3,19 +3,14 @@
   "private": true,
   "scripts": {
     "serve": "vue-cli-service serve",
-    "build": "vue-cli-service build",
-    "lint": "vue-cli-service lint"
+    "build": "vue-cli-service build"
   },
   "dependencies": {
     "vue": "^3.3.4"
   },
   "devDependencies": {
-    "@vue/cli-plugin-babel": "^4.4.0",
-    "@vue/cli-plugin-eslint": "^4.4.0",
-    "@vue/cli-service": "^4.4.0",
-    "babel-eslint": "^10.1.0",
-    "eslint": "^6.7.2",
-    "eslint-plugin-vue": "^6.2.2",
-    "vue-template-compiler": "^2.6.11"
+    "core-js": "^3.32.2",
+    "@vue/cli-plugin-babel": "^5.0.8",
+    "@vue/cli-service": "^5.0.8"
```

Comment:
I like that the dependencies get quite simpler.

Note that the project generated by `@vue/cli@5` put `core-js` in the `dependencies`, and I moved it to `devDependencies` because as far as I am aware it is only used by the Babel config used by `@vue/cli-plugin-babel` to inject ECMAScript polyfills at build time. We don't want to download it as a dependency when running in EngineBlock.